README.md

```diff
@@ -59,3 +64,51 @@ func main() {
 	}
 }
 ```
+
+### HTTP
+
```

Comment:
nit: would be great a doc link somewhere here. This would be helpful for users who don't know that we have ILP/TCP and ILP/HTTP and Go client supports both of them.
https://questdb.io/docs/reference/api/ilp/overview/#http-and-tcp-overview